src/styles/theme/oc-button.scss

```diff
@@ -20,7 +20,8 @@
 
 .oc-button {
   @extend .uk-button;
-  @extend .uk-position-relative;
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5442879)